### PR TITLE
MRG: Allow skipping topology check

### LINKED ISSUE #4873: Skipping dense skin surface topology check
For reasons I don't really understand nor can clearly spot in MRI data the high-density skin surface routine `make_scalp_surfaces` fails b/c the srf topology is botched and causes `_assert_complete_surface` to throw...  
>RuntimeError: Surface outer skin is not complete (sum of solid angles = 6.2829 * 4*PI instead).

I've tried `--force` and cutting off the neck but still hitting this more frequnetly than I'd like.
So I thought a PR to skip the skin surface topology check might be useful?

### PR DESCRIPTION
**Pull request by larsoner**
Closes #4873.

@ktavabi can you see if this fixes your problem? Run with `-i` or `--allow-incomplete`.

### CODE REVIEW

File: mne/commands/mne_make_scalp_surfaces.py [hunk 1]
```diff
@@ -49,20 +49,25 @@ def run():
     parser.add_option("-n", "--no-decimate", dest="no_decimate",
                       help="Disable medium and sparse decimations "
                       "(dense only)", action='store_true')
+    parser.add_option("-i", "--allow-incomplete", dest="allow_incomplete",
+                      help="Allow the head surface to be 'incomplete' "
+                      "according to the sum of solid angles check.",
+                      action='store_true')
```
Comment (agramfort) [inline mne/commands/mne_make_scalp_surfaces.py]: it's not a dupe of the --force option?

Comment (larsoner) [reply mne/commands/mne_make_scalp_surfaces.py]: ahh yes it is. And it looks like `force` is never currently used

**Comment by larsoner** [comment]
Fixed, @ktavabi use `--force`

**Comment by ktavabi** [comment]
Same error futher down stream in the rouitine...
`mne make_scalp_surfaces --overwrite --subject sub --force`

> 1. Creating a dense scalp tessellation with mkheadsurf...
> 2. Creating /Users/ktavabi/Data/freesufer/sub-nbwr144_ses-1_fsmempr_ti1100_rms_1_freesurf_hires_nochop/bem/sub-nbwr144_ses-1_fsmempr_ti1100_rms_1_freesurf_hires_nochop-head-dense.fif ...
> outer skin  CM is   0.32   5.41 -30.07 mm
> /Users/ktavabi/Projects/Work/mne-python/mne/commands/mne_make_scalp_surfaces.py:130: RuntimeWarning: Surface outer skin  is not complete (sum of solid angles yielded 0.999955, should be 1.)
>   run()
> Surface outer skin  is not complete (sum of solid angles yielded 0.999955, should be 1.)
> Surfaces passed the basic topology checks.
> 3. Creating medium tessellation...
> 3.1 Decimating the dense tessellation...
> 3.2 Creating /Users/ktavabi/Data/freesufer/sub-nbwr144_ses-1_fsmempr_ti1100_rms_1_freesurf_hires_nochop/bem/sub-nbwr144_ses-1_fsmempr_ti1100_rms_1_freesurf_hires_nochop-head-medium.fif
> outer skin  CM is   1.82  10.51 -40.92 mm
> Traceback (most recent call last):
>   File ".../mne/commands/mne_make_scalp_surfaces.py", line 130, in <module>
>     run()
>   File ".../mne/commands/mne_make_scalp_surfaces.py", line 60, in run
>     options.no_decimate, options.verbose)
>   File "<string>", line 2, in _run
>   File "...mne-python/mne/utils.py", line 728, in verbose
>     return function(*args, **kwargs)
>   File ".../mne_make_scalp_surfaces.py", line 125, in _run
>     [mne.io.constants.FIFF.FIFFV_BEM_SURF_ID_HEAD], [1], rescale=False)
>   File ".../mne/bem.py", line 502, in _surfaces_to_bem
>     _check_surfaces(surfs, incomplete=incomplete)
>   File ".../mne/bem.py", line 438, in _check_surfaces
>     _assert_complete_surface(surf, incomplete=incomplete)
>   File ".../mne/bem.py", line 413, in _assert_complete_surface
>     raise RuntimeError(msg)
> RuntimeError: Surface outer skin  is not complete (sum of solid angles yielded 0.999941, should be 1.)

**Comment by larsoner** [comment]
Okay that one should be fixed now, too

**Comment by ktavabi** [comment]
LGTM. Thanks guys.

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**